src/Amadevus.RecordGenerator.Generators/EqualityGenerators.cs

```diff
@@ -154,14 +154,14 @@ private static MemberDeclarationSyntax GenerateGetHashCode(RecordDescriptor desc
         {
             const string varKeyWord = "var";
             const string hashCodeVariableName = "hashCode";
+
```

Comment:
Additional empty line not needed. There are a couple other places with additional empty lines that can be removed (within a method body).

Reply:
I prefer it with line breaks to make the code easier to read.  Write once, read many.